gpm_api/tests/test_visualization/test_plot.py

```diff
@@ -203,14 +203,24 @@ def test_orbit_nan_along_track(
         p = plot.plot_map(orbit_nan_along_track_dataarray)
         save_and_check_figure(figure=p.figure, name=get_test_name())
 
-    def test_orbit_nan_lon_cross_track(
+    def test_orbit_nan_one_cell(
+        self,
+        orbit_dataarray: xr.DataArray,
+    ) -> None:
+        """Test plotting orbit data with NaN values at one cell"""
+
+        orbit_dataarray.data[2, 2] = np.nan
+        p = plot.plot_map(orbit_dataarray)
+        save_and_check_figure(figure=p.figure, name=get_test_name())
+
+    def test_orbit_nan_lon_cross_track(  # TODO: check behavior
```

Comment:
The figure from this test looks bad. 
But in the definition of  `orbit_nan_lon_cross_track_dataarray`, you used `cross_track_index = 1` which is not the outer cross-track ... if you use `cross_track_index = 0` the bad behaviour does not occur I guess. Can you try out? 
If the figures looks correct, please rename this test as `test_orbit_invalid_outer_cross_track_coordinates` (for `cross_track_index = 0`)

But I guess you exposed a bug in the interpolation of invalid coordinates ... so can you open an issue with a reproducible example of this plot? I can try to fix it tomorrow or Friday after this big PR is merged.

Reply:
#41